packages/model-registry/upstream/frontend/docs/dev-setup.md

```diff
@@ -5,7 +5,7 @@
 This project requires the following tools to be installed on your system:
 
 - [NodeJS and NPM](https://nodejs.org/)
-  - Node recommended version -> `20.17.0`
+  - Node recommended version -> `22.17.0`
```

Comment:
Don't change this. Wait for the update from upstream.

Reply:
Agh, yeah, I missed this one, thanks

Reply:
Fixed!